include/seqan3/range/container/bitcompressed_vector.hpp

```diff
@@ -125,12 +131,21 @@ class bitcompressed_vector
         //!\brief The proxy of the underlying data type; wrapped in semiregular_t, because it isn't semiregular itself.
         ranges::semiregular_t<internal_proxy_type> internal_proxy;
 
+        //!\brief Update the sdsl-proxy.
+        constexpr void impl()
```

Comment:
why not call it `update_proxy`? or `propagate_value`?

This seems to be an event, in different languages I would write `on_update` xD

Reply:
> why not call it update_proxy?

it doesn't normally update another proxy, but the underlying data structure.

> This seems to be an event, in different languages I would write on_update 

I like `on_update()`! I will change it!